cmd/kiosk/main.go

```diff
@@ -9,43 +9,194 @@ import (
 
 	"github.com/SebastianRau/kiosk/pkg/kiosk"
 	"github.com/ilyakaznacheev/cleanenv"
+	"github.com/palantir/go-encrypted-config-value/encryptedconfigvalue"
 )
 
 var (
-	EXIT_NO_CONFIG   = 1
-	EXIT_NO_URL      = 2
-	EXIT_INVALID_URL = 3
+	EXIT_NO_CONFIG             = 1
+	EXIT_NO_URL                = 2
+	EXIT_INVALID_URL           = 3
+	EXIT_KEY_GENERATION_FAIL   = 4
+	EXIT_PRIVATE_KEY_NOT_FOUND = 5
 )
 
 var (
 	version = "0.0.0xxx"
 )
 
+func setEnvironment() {
+	// for linux/X display must be set
+	var displayEnv = os.Getenv("DISPLAY")
+	if displayEnv == "" {
+		log.Println("DISPLAY not set, autosetting to :0.0")
+		if err := os.Setenv("DISPLAY", ":0.0"); err != nil {
+			log.Println("Error setting DISPLAY", err.Error())
+		}
+		displayEnv = os.Getenv("DISPLAY")
+	}
```

Comment:
okay it is just moved code but what is the benefit of the extra mile of setting an environmental variable? Would't it be sufficient to just set `displayEnv = ":0.0"` directly as a fall back value inside the if-block?